test/unit/create_task.spec.js

```diff
@@ -4,13 +4,13 @@ import CreateTask from '../../src/app/components/task/CreateTask';
 
 
 describe('<CreateTask />', () => {
-  const media_archived = { archived: true , permissions:"{\"create Task\": true }" };
-  const media_not_archived = { archived: false, permissions:"{\"create Task\": true }" };
+  const media_archived = { archived: 1 , permissions:"{\"create Task\": true }" };
+  const media_not_archived = { archived: 0, permissions:"{\"create Task\": true }" };
```

Comment:
Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

Reply:
Fixed by using CheckArchivedFlags constants